### ISSUE TITLE
Incorrect implicit valid usage for VkCudaLaunchInfoNV

### ISSUE THREAD

**Issue by expipiplus1** [KhronosGroup/Vulkan-Docs#2261]
The Valid Usage for VkCudaLaunchInfoNV:pParams specifies that `[...] pParams must be a valid pointer to an array of paramCount bytes`, when it should be a `valid pointer to an array of paramCount pointers`

This occurs in at least the following valid usage specs concerning `void**` values.

```
VUID-VkCudaLaunchInfoNV-pParams-parameter
VUID-VkCudaLaunchInfoNV-pExtras-parameter
VUID-VkCuLaunchInfoNVX-pParams-parameter
VUID-VkCuLaunchInfoNVX-pExtras-parameter
```

**Comment by oddhack** [comment]
That looks like a logic error in the validity generator script and affects a couple of other places as well - good catch! I'll take a look at it but expect this might be complex enough to take a while to sort out.

**Comment by pdaniell-nv** [comment]
Pending fix is here: https://gitlab.khronos.org/vulkan/vulkan/-/merge_requests/6382

**Comment by pdaniell-nv** [comment]
The fix was merged to [1.3.175](https://github.com/KhronosGroup/Vulkan-Docs/releases/tag/v1.3.275). Closing.
